src/fsharp/NameResolution.fs

```diff
@@ -1541,12 +1541,11 @@ type TcResultsSinkImpl(g, ?source: string) =
                 let positions =
                     [|
                         yield 0
-                        for i in 0..source.Length-1 do
-                            let c = source.[i]
-                            if c = '\r' && i + 1 < source.Length && source.[i+1] = '\n' then ()
-                            elif c = '\r' then yield i + 1
-                            if c = '\n' then yield i + 1
-                        yield source.Length
+                        for i in 1..source.Length do
+                            let c = source.[i-1]
+                            if c = '\r' && i < source.Length && source.[i] = '\n' then ()
+                            elif c = '\r' then yield i
+                            elif c = '\n' then yield i
                     |]
```

Comment:
@AviAvni.   What happens if source is a string with no '\r'  or '\n'.  I think it should yield source.Length.  However, this doesn't.  Or is that unnecessary?

Reply:
It's yield source.Lemgth one line later

Reply:
Wierdly enough, it showed that line as deleted on my machine at work.  But yes, I see it now.

Thx.